shell/browser/api/atom_api_top_level_window.cc

```diff
@@ -246,6 +246,10 @@ void TopLevelWindow::OnWindowSwipe(const std::string& direction) {
   Emit("swipe", direction);
 }
 
+void TopLevelWindow::OnWindowRotate(const float rotation) {
+  Emit("rotate", rotation);
```

Comment:
`rotate` doesn't imply the right thing here, it could be read as the window rotating.  This should probably be called `rotate-gesture` or `swipe-rotate` or something that makes it more clear it's a finger gesture

Reply:
good point!

Reply:
switched `rotate` -> `rotate-gesture` both in the exposed API and the underlying native code for clarity's sake!